src/utils/api.ts

```diff
@@ -41,9 +41,13 @@ const authConfig = {
   grant_type: "client_credentials",
 };
 
-const constructApiHeaders = (token = "", contentType = "application/json") => ({
+/**
+ * constructApiHeaders
+ * Creates the authorization header to use when calling the NYPL Platform API.
+ */
+export const constructApiHeaders = (token: string) => ({
   headers: {
-    "Content-Type": contentType,
+    "Content-Type": "application/json",
```

Comment:
Why the change?

Reply:
It was set as that value for the default but no other value was passed and no other value will be passed, so it's okay to hardcode it.